blocks/large-manual-promo-block/features/large-manual-promo/default.jsx

```diff
@@ -65,20 +65,24 @@ const LargeManualPromo = ({ customFields }) => {
 		) : null;
 	};
 
-	const PromoHeading = () =>
-		showHeadline ? (
-			<Heading>
-				<Conditional
-					component={Link}
-					condition={linkURL}
-					href={formatURL(linkURL)}
-					openInNewTab={newTab}
-					onClick={registerSuccessEvent}
-				>
-					{headline}
-				</Conditional>
-			</Heading>
-		) : null;
+	const PromoHeading = () => {
+		if (showHeadline) {
+			return (
+				<Heading>
+					<Conditional
+						component={Link}
+						condition={linkURL}
+						href={formatURL(linkURL)}
+						openInNewTab={newTab}
+						onClick={registerSuccessEvent}
+					>
+						{headline}
+					</Conditional>
+				</Heading>
+			);
+		}
+		return null;
+	};
```

Comment:
This change doesn't change any logic, it's just a rewrite of what was there  The actual fix would be:
```suggestion
	const PromoHeading = () =>
		showHeadline && headline ? (
			<Heading>
				<Conditional
					component={Link}
					condition={linkURL}
					href={formatURL(linkURL)}
					openInNewTab={newTab}
					onClick={registerSuccessEvent}
				>
					{headline}
				</Conditional>
			</Heading>
		) : null;
```

Wouldn't it?